From 4db0753959dc7aeff567c689be6c4eb611aea8ee Mon Sep 17 00:00:00 2001
From: Martin Liska <mliska@suse.cz>
Date: Mon, 24 Oct 2022 15:34:39 +0200
Subject: [PATCH 165/547] x86: fix VENDOR_MAX enum value

	PR target/107364

gcc/ChangeLog:

	* common/config/i386/i386-cpuinfo.h (enum processor_vendor):
	  Reorder enum values as BUILTIN_VENDOR_MAX should not point
	  in the middle of the valid enum values.

(cherry picked from commit f751bf4c5d1aaa1aacfcbdec62881c5ea1175dfb)
---
 gcc/common/config/i386/i386-cpuinfo.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/gcc/common/config/i386/i386-cpuinfo.h b/gcc/common/config/i386/i386-cpuinfo.h
index 3f6d2013daf..0f12024141c 100644
--- a/gcc/common/config/i386/i386-cpuinfo.h
+++ b/gcc/common/config/i386/i386-cpuinfo.h
@@ -33,8 +33,10 @@ enum processor_vendor
   VENDOR_CENTAUR,
   VENDOR_CYRIX,
   VENDOR_NSC,
+
+  /* Maximum values must be at the end of this enum.  */
+  VENDOR_MAX,
   BUILTIN_VENDOR_MAX = VENDOR_OTHER,
-  VENDOR_MAX
 };
 
 /* Any new types or subtypes have to be inserted at the end. */
-- 
2.25.1

